### PR TITLE
Make script changes to support openshift

### LINKED ISSUE #5: Add support for openshift
Code changes and script updates to deploy kruize to openshift

### PR DESCRIPTION
**Pull request by ShishirH**
Closes #5 
1. Removed nodeSelector and tolerations from template yaml
2. Renamed minikube manifest folder to serviceMonitor
3. Modified script to apply serviceMonitor for openshift

Signed-off-by: Shishir Halaharvi <shihala1@in.ibm.com>

### CODE REVIEW

**Comment by ShishirH** [comment]
Tested on openshift setup.

File: deploy.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ GRAFANA_DOCKER_IMAGE="grafana/grafana:latest"
 KRUIZE_DOCKER_REPO="dinogun/kruize"
 KRUIZE_DOCKER_IMAGE=${KRUIZE_DOCKER_REPO}:${KRUIZE_VERSION}
 
-MINIKUBE_MANIFEST="manifests/minikube/kruize-service-monitor.yaml"
+SERVICE_MONITOR_MANIFEST="manifests/serviceMonitor/kruize-service-monitor.yaml"
```
Comment (dinogun) [inline deploy.sh]: No camel casing for script names/dirs please. `servicemonitor` should be fine.

File: deploy.sh [hunk 2]
```diff
@@ -408,10 +410,6 @@ function openshift_deploy() {
 }
 
 function openshift_start() {
-	echo
-	echo "OpenShift support coming soon!"
-	exit -1;
-
```
Comment (dinogun) [inline deploy.sh]: Need to be consistent for all installs
```
    echo
    echo "###   Installing kruize for OpenShift"
    echo
```

Comment (dinogun) [reply deploy.sh]: Need to add the warning as above and ask for confirmation from user to proceed. For automated mode, warning still needs to be printed but no need for confirmation.

Comment (ShishirH) [reply deploy.sh]: Should the warning message and the confirmation be in function `openshift_start` or `openshift_prereq` (like minikube)?

File: deploy.sh (hunk 1 above)
Comment (ShishirH) [reply deploy.sh]: Changed dir name.

File: deploy.sh (hunk 2 above)
Comment (dinogun) [reply deploy.sh]: It is good to do it in `_start`, that way no damage done if the user chooses 'no'.

Comment (ShishirH) [reply deploy.sh]: Added warning and confirmation

**Comment by ShishirH** [comment]
@dinogun `openshift_first` now takes in all parameters when in non-interactive mode

**Pull request merged**